rspec-mode.el

```diff
@@ -635,6 +651,15 @@ file if it exists, or sensible defaults otherwise."
          (substring file (length remote))
        file))))
 
+(defun rspec--vagrant-wrapper (command)
+  (if (rspec-vagrant-p)
+      (format "vagrant ssh -c 'cd %s; %s'"
+              rspec-vagrant-cwd
+              (replace-regexp-in-string (rspec-project-root)
+                                        rspec-vagrant-cwd
```

Comment:
But the result is already `shell-quote-argument`-ed, right?

Anyway, if you do it in `rspec--shell-quote-local`, that won't be necessary.